pinot-common/src/main/java/org/apache/pinot/common/utils/config/QueryOptionsUtils.java

```diff
@@ -241,6 +234,13 @@ public static Integer getMinBrokerGroupTrimSize(Map<String, String> queryOptions
     return uncheckedParseInt(QueryOptionKey.MIN_BROKER_GROUP_TRIM_SIZE, minBrokerGroupTrimSizeString);
   }
 
+  @Nullable
+  public static Integer getMSEMinGroupTrimSize(Map<String, String> queryOptions) {
```

Comment:
nit: we don't enforce that (probably there is no way to do that) but that's not the allowed way to camel case acronymous in Google Style Guide (section [5.3](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)). 

Anyway, it is just a nit.

Reply:
I see. Unfortunately we are already using `MSE` this way in a lot of places, so I'd prefer keeping them consistent.